CHANGES.txt

```diff
@@ -5,6 +5,9 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Fixed a rare race condition that could cause some queries to return
```

Comment:
maybe add in the changes that this affected only queries with a limit/offset > 1_000_000